lib/moog.js

```diff
@@ -474,7 +474,7 @@ module.exports = function(options) {
 
   return self;
 
-  function createPrep(className) {
+  async function createPrep(className) {
```

Comment:
I see that we annotate each definition with its own `__meta` property, which seems to spell trouble for multisite like I said, but in `moog.js` we also replace every definition with a shallow clone of itself and clone `__meta` too, so this may be OK. But if it's OK, why did we use `importFresh`? :thinking: 

It'll need testing in multisite in projects with two or more themes with different modules enabled and sites of both themes active.

Reply:
importFresh is now back as dynamic import expression

Reply:
👍